### ISSUE TITLE
Please read first

### ISSUE THREAD

**Issue by cztomsik** [cztomsik/graffiti#194]
Hello and thank you for your interest in the project.

- If you want to ask question, the best place currently is our [Discord server](https://discord.gg/zQwyzFb)
- If you want to report bug, please wait until the first release (or feel free to [ask in chat](https://discord.gg/zQwyzFb))
- If you have feature request, then unfortunately, the answer is no, unless you want to work on that yourself and then we should [have a chat first](https://discord.gg/zQwyzFb)

So basically, go to discord and keep this area private for me (for now). Thank you.
